testing/api/test_api.py

```diff
@@ -1249,6 +1571,30 @@ def test_get_profiles_format(testrun):  # pylint: disable=W0613
     assert "question" in item
     assert "type" in item
 
+@responses.activate
+def test_get_profiles_format_internal_server_error(testrun): # pylint: disable=W0613
+  """Test for get_profiles_format causing internal server error"""
+
+  # Mock the response for GET request for getting profiles format
+  responses.add(
```

Comment:
Don't mock the API itself, either mock the session object or load Testrun with an invalid profile format JSON

Reply:
As discussed, Testrun should not start if the profiles format fails to load. We will remove this test and implement the correct code for the session when this occurs.